.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this ECLIPSE project file be here?

Reply:
no. weird. Thanks!